packages/DraftEditor/src/text-block-parser/is-valid-json.tsx

```diff
@@ -0,0 +1,9 @@
+function isValidJSON(candidate: any) {
+  try {
+    return JSON.parse(candidate);
+  } catch (e) {
```

Comment:
Do you need to catch the event here?

Reply:
is ''type script lint'' in the catch the error must be defined even if you don't use it

Reply:
Maybe we could change that rule if it's not useful anymore